sidebars.js

```diff
@@ -40,7 +40,9 @@ module.exports = {
             }
         ],
         Simulate: ["simulation/simulation"],
-        "Analyze Tab": ["analysis/variance-analysis"],
+        "Analyze Tab": [
+            "analysis/Analyze tab"
```

Comment:
I think we can call it as **Variance Analysis**

like 
```suggestion
        "Analyze Tab": [
            "analysis/Variance Analysis"
```